ci_scripts/ci-prelude.sh

```diff
@@ -30,49 +22,15 @@ if [ -z "$TRAVIS_OS_NAME" ]; then
 fi
 SYS_DC_CORE=${SYS_DC_CORE:-true}
 
+export PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig
 
-case $TRAVIS_OS_NAME in
-    linux)
-        docker pull $DOCKER_IMAGE
-        CONTAINER_ID=$(docker run -d -v/etc/passwd:/etc/passwd:ro -u$(id -u):$(id -g) -v$(pwd):/work -w/work -eHOME=/work $DOCKER_IMAGE)
-        EXEC="docker exec $CONTAINER_ID";
-        EXEC_ROOT="docker exec -u0:0 -eHOME=/ $CONTAINER_ID";
-        if [ "$SYS_DC_CORE" = "true" ]; then
-            $EXEC git clone --branch=$DC_CORE_VERSION https://github.com/deltachat/deltachat-core deltachat-core-src
-            $EXEC meson -Drpgp=true deltachat-core-build deltachat-core-src
-            $EXEC ninja -v -C deltachat-core-build
-            $EXEC_ROOT ninja -v -C deltachat-core-build install
-            $EXEC_ROOT ldconfig -v
-            $EXEC rm -rf deltachat-core-build deltachat-core-src
-        fi
-        ;;
-    osx)
-        # Install cyrus sasl
-        ./ci_scripts/shared-image-context/build-install-sasl.sh --with-openssl=/usr/local/opt/openssl
-
-        export PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig
-        
-        # Install rust
-        curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs \
-            | sh -s -- --default-toolchain nightly -y
-        . ~/.cargo/env
+# Install rust
+chmod +x ./ci_scripts/shared-prelude-context/install-rust.sh
```

Comment:
git is supposed to keep permissions so this shouldn't be needed normally?  not that it does much harm.

Reply:
Good idea!